### ISSUE TITLE
Option to disable specific UI modules

### ISSUE THREAD

**Issue by sandervspl** [Karl-HeinzSchneider/WoW-DragonflightUI#136]
Sorry for the many issues! After today another compat issue with leatrix plus. As you've mentioned before it is difficult to have full compat with LTP, so my question is if it would be possible to add toggles for each UI element? For example, the questlog was updated, but I like using the enhanced quest log from LTP as I find the regular quest log to be quite unusable.

![image](https://github.com/user-attachments/assets/ebf8e3e9-af9d-41fc-a61a-f647b1067ebd)

**Comment by sandervspl** [comment]
I see trainers/professions already have a toggle. Quest log is missing

**Comment by sandervspl** [comment]
My quest log is fixed now after a reload... Not sure what the issue was

**Comment by Karl-HeinzSchneider** [comment]
Deactivate the LeatrixPlus Questlog option; I think a chat message should pop up if you have this enabled.

**Comment by Karl-HeinzSchneider** [comment]
I made the Era Questlog exactly like the new Cata one, so you dont need any other addon/Leatrix etc.  (The default Era Questlog is indeed unuseable)

My goal is to do a complete overhaul mod, so the default frames etc should be good already. Other, bigger addons can improve that, but small tweaks like some of the more popular options from addons like Leatrix will be implemented as default.

Some Era frames need a complete overhaul because the 20 year old blizzard code is so bad and inconsistent. They are made with custom textures for simple things like the frame border style etc, while the newer frame use a Template with a nineslice texture. So you cant just swap the textures, because I would have to draw them myself for so many frames. 

With my UI system I can add the frame style with 3 lines of code, and they are then consistent with all the other frames. Most addons that hook onto some of the old frames, like the addon compatibility I did in the last few days, are also not coded in a good way, because a lot of stuff is hardcoded, and any change to the original frame breaks it completely, although there would easier ways to add onto them.
